zombies.js

```diff
@@ -110,8 +144,17 @@
  * @param {Item/Weapon/Food} item   The item to take.
  * @return {boolean} true/false     Whether player was able to store item in pack.
  */
-
-
+ takeItem(item) {
+  let packContent = this._pack;
+  if (packContent.length >= 3 && !packContent.includes(item)) {
```

Comment:
this is saying the pack has to have 3 or more items *and* the pack can not include item.
I dont think the second part of this test in necessary.